src/olympia/versions/models.py

```diff
@@ -645,6 +645,21 @@ def was_auto_approved(self):
         return False
 
 
+class VersionPreview(BasePreview, ModelBase):
+    version = models.ForeignKey(Version, related_name='previews')
+    position = models.IntegerField(default=0)
+    sizes = JSONField(max_length=25, default={})
+    media_folder = 'version-previews'
+
+    class Meta:
+        db_table = 'version_previews'
+        ordering = ('-version', 'position', 'created')
+
+
+dbsignals.pre_save.connect(save_signal, sender=VersionPreview,
+                           dispatch_uid='version_preview_translations')
```

Comment:
Oh, actually we don't need this because I didn't include captions for VersionPreview.  @diox should I for future requirements or do that if/when needed?

Reply:
Was about to suggest removing it :) We'll add it back if we ever need it, let's remove for now. Happy to see caption-less previews btw :smile: